_posts/2022-06-03-timo-tracepoints-monitoring.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Timo – Tracepoints and monitoring the Bitcoin network"
```

Comment:
People usually know me as 0xB10C

```suggestion
title: "0xB10C – Tracepoints and monitoring the Bitcoin network"
```

Reply:
Right, I replaced the instances of "Timo" throughout with 0xB10C